assets/src/styles/editorOverrides.scss

```diff
@@ -213,3 +213,18 @@ input.describe[type=text][data-setting=caption] {
   font-size: 13px;
   color: red;
 }
+
+// Articles block overrides
```

Comment:
Maybe we can include these change in assets/src/styles/blocks/ArticlesEditor.scss too?

Reply:
Indeed 👍